js/selectors/report-tree.js

```diff
@@ -42,6 +42,9 @@ const isQuestionVisible = (question, featuredOnly) => {
   return true;
 };
 
+// allow for anonymous students
+const getStudentId = (answer) => answer.get("platformUserId") || answer.get("runKey");
```

Comment:
This makes sense, but instead of doing this, LARA should set the platform_user_id field with the value of the run_key when it saves the answers.  That is what the AP does. This way tools that are working with the firestore answers don't need to care as much about the run_key and can just treat the platform_user_id (plus platform_id if it is exists) as the unique identifier for the student.

Here is a working AP runKey based report:
https://portal-report.concord.org/branch/master/index.html?runKey=f671cd0c-75eb-4ceb-98ba-d3ea5ae3d10c&activity=https://authoring.staging.concord.org/activities/20874&resourceUrl=https://authoring.staging.concord.org/activities/20874&firebase-app=report-service-dev&sourceKey=authoring.staging.concord.org&answersSourceKey=activity-player.concord.org

This is what one of the answers looks like in firestore from in that report:
![image](https://user-images.githubusercontent.com/86016/134346942-d3f45b81-e30e-430c-893a-a4fa4ff8b3a7.png)